Sources/StreamChatUI/CommonViews/ChatMessageQuoteView/ChatMessageQuoteView.swift

```diff
@@ -0,0 +1,224 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The `ChatMessageQuoteView` content.
+public struct ChatMessageQuoteViewContent<ExtraData: ExtraDataTypes> {
+    /// The quoted message.
+    let message: _ChatMessage<ExtraData>?
+    /// The avatar position in relation with the text message.
+    let avatarAlignment: ChatMessageQuoteAvatarAlignment
+}
```

Comment:
I think this is a pattern we could adopt. I think using tuples for the content of the views is a bit limiting and not very scalable if the tuple starts to grow. Also, auto-completion with tuples is worse than using a regular type. WDYGT?

Reply:
The problem with the Nested type inside the `ChatMessageQuoteView` is that it makes it harder to type because of ExtraData 😕 In order to access the subtype, the customer will have to write `ChatMessageQuoteView<ExtraData>.Content`. Probably at first, you will write just `ChatMessageQuoteView.Content` and the compiler will warn you that this is not possible, if the customer knows about ExtraData, he will be able to tell what is wrong, but is not very intuitive.